File: backend/Models/Point.cs
namespace Graphic_Neural_Network.backend.Models
{
    public class Point
    {
        public int Id { get; set; }
        public double X { get; set; }
        public double Y { get; set; }
        public int Class { get; set; }

        public Point(int id, double posX, double posY, int piontClass)
        {
            Id = id;
            X = posX;
            Y = posY;
            Class = piontClass;
        }
    }
}

File: frontend/src/models/data/Point.ts

export default class Point {
    x: number;
    y: number;
    class: number;
    expectedOutput: number[] = [];

    constructor(posX: number, posY: number, classNr: number, expectedOutput: number[]) {
        this.x = posX;
        this.y = posY;
        this.class = classNr;
        this.expectedOutput = expectedOutput;
    }
}